### PR TITLE
GitHub on push webhook action

### PR DESCRIPTION
**Pull request by EdenComp**
Also fixes ARE-309
Also fixes ARE-287

### CODE REVIEW

**Review by Tomi-Tom — APPROVED** [review]

File: backend/back/src/database.ts [hunk 1]
```diff
@@ -92,6 +93,7 @@ export const DATA_SOURCE_OPTIONS: DataSourceOptions = {
 		CreateIssueGithubReaction1698062010534,
 		CreateGoogleSlideInPresentationArea1698150990404,
 		UpdateServiceImagesToBrandFetchSVGs1698161840408,
+		OnIssueCreatedGithubAction1698179108023,
```
Comment (Croos3r) [inline backend/back/src/database.ts:96]: Please follow the name convention for the migrations "Create<YourAreaName>Area"

**Review by RezaRahemtola — APPROVED** [review]

File: backend/back/package.json [hunk 2]
```diff
@@ -42,6 +42,7 @@
     "axios": "^1.5.1",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
+    "crypto": "^1.0.1",
```
Comment (Croos3r) [inline backend/back/package.json:45]: I don't understand why you are putting this in the package.json, this is now a builtin: https://www.npmjs.com/package/crypto

File: backend/back/src/webhooks/webhooks.module.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { Module } from "@nestjs/common";
+import { WebhooksController } from "./webhooks.controller";
+import { ConfigModule } from "@nestjs/config";
+import { GithubWebhookService } from "./services/gihub-webhook.service";
+import { GrpcModule } from "../grpc/grpc.module";
+
+@Module({
+	imports: [ConfigModule.forRoot(), GrpcModule],
```
Comment (Croos3r) [inline backend/back/src/webhooks/webhooks.module.ts:8]: The config module is global you don't have to import it and surely not with forRoot

**Review by Croos3r — CHANGES_REQUESTED** [review]

File: backend/back/src/database.ts (hunk 1 above)
Comment (EdenComp) [reply backend/back/src/database.ts:96]: It doesn't really make sense for actions

Comment (Croos3r) [reply backend/back/src/database.ts:96]: why ?

File: backend/back/package.json (hunk 2 above)
Comment (EdenComp) [reply backend/back/package.json]: Fixed

File: backend/back/src/webhooks/webhooks.module.ts (hunk 3 above)
Comment (EdenComp) [reply backend/back/src/webhooks/webhooks.module.ts]: Fixed

File: backend/back/src/database.ts (hunk 1 above)
Comment (EdenComp) [reply backend/back/src/database.ts:96]: You want me to call it CreateOnIssueGithubAction ???

Comment (Croos3r) [reply backend/back/src/database.ts:96]: CreateGithubOnIssueCreationArea

Comment (EdenComp) [reply backend/back/src/database.ts]: Done

**Review by Croos3r — APPROVED** [review]
LGTM

**Pull request merged**